frontend/src/components/DropdownMenu.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import './DropdownMenu.css';
+import triangleDown from '../images/triange_down.png';
+
+export default function DropdownMenu({
```

Comment:
takes a lot of props as button select did but will hopefully be a little easier to work with... also, many of these are optional